helix-core/src/main/java/org/apache/helix/controller/stages/BestPossibleStateCalcStage.java

```diff
@@ -259,7 +259,7 @@ private boolean computeResourceBestPossibleState(ClusterEvent event, ResourceCon
         return checkBestPossibleStateCalculation(idealState);
       } catch (Exception e) {
         LogUtil
-            .logError(logger, _eventId, "Error computing assignment for resource " + resourceName + ". Skipping.", e);
+            .logError(logger, _eventId, "Error computing assignment for resource " + resourceName + ". Skipping.");
         // TODO : remove this part after debugging NPE
```

Comment:
Can we remove this piece of code (based on the TODO comment)?

Reply:
I think we can remove the part after TODO.